docs/pages/repo/docs/core-concepts/monorepos/running-tasks.mdx

```diff
@@ -263,63 +159,21 @@ be explicitly opted into via including `//#<task>` in the pipeline configuration
 some best-effort checking to produce an error in the recursion situations, but it is up to you to only
 opt in those tasks which don't themselves trigger a `turbo` run that would recurse.
 
-### Dependencies outside of a task
-
-When your task has topological dependencies that are outside of that given task, you'll still want to enjoy the parallelism of Turborepo and ensure that your caching behavior is correct according to your code changes.
-
-To demonstrate how to do this, let's say you have a set of workspaces to do a little bit of math: `add`, `subtract`, and `multiply`. `subtract` is implemented by calling `add` with a negative number and your `multiply` works by calling `add` in a loop. So, `add` is a dependency of both `subtract` and `multiply`.
-
-You've written tests in all three of these workspaces and it's time to run them. There are two requirements here:
-
-1. All tests run in parallel to keep things speedy
-2. A change in a dependency should result in a cache miss
-
-To accomplish this, we can set up a pipeline like so:
-
-```jsonc filename="turbo.json"
-{
-   "$schema": "https://turbo.build/schema.json",
-   "pipeline": {
-     "topo": {
-       "dependsOn": ["^topo"]
-     },
-     "test": {
-       "dependsOn": ["^topo"]
-     }
-   }
- }
-```
-
-![](/images/docs/task-graph-with-placeholder-task.png)
-
-In this pipeline, we create an intermediary placeholder `topo` task. Since we don't have a `topo` command in our workspaces, the pipeline will go straight to running `test` scripts in parallel, meeting our first requirement. The second requirement will also be taken care of, falling back on Turborepo's default behavior of creating hashes for a workspace task and it's dependencies as a tree.
-
-<Callout
-  type="idea"
-  icon={<HeartIcon className="h-5 w-5 mt-1 text-gray-400" aria-hidden="true" />}
->
-  Turborepo's Pipeline API design and this page of documentation was inspired by
-  [Microsoft's Lage
-  project](https://microsoft.github.io/lage/docs/Tutorial/pipeline#defining-a-pipeline).
-  Shoutout to [Kenneth Chau](https://twitter.com/kenneth_chau) for the idea of
-  fanning out tasks in such a concise and elegant way.
-</Callout>
-
 ### Incremental Adoption
 
 After you've declared a task in `turbo.json`, it's up to you to implement it in
 your `package.json` manifests. You can add scripts all at once, or one workspace
 at at a time. Turborepo will gracefully skip workspaces that don't include the
 task in their respective package.json manifest.
 
-For example, if your repository has three workspaces:
+For example, if your repository has the three workspaces (similar to the ones mentioned above on this page):
 
 ```bash
 apps/
   web/package.json
   docs/package.json
 packages/
-  ui/package.json
+  shared/package.json
```

Comment:
There's still a bunch of references to `ui` in this section that need to be updated to `shared`